roles/bootstrap-os/tasks/main.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- name: Fetch /etc/os-release
+  raw: cat /etc/os-release
+  register: os_release
+  changed_when: false
+
+- name: Set bootstrap_os
+  set_fact:
+    bootstrap_os: >-
```

Comment:
coul you rename `bootstrap_os` to `os_family`  
(I've just changed it in https://github.com/kubernetes-incubator/kubespray/commit/c27a91f7f050c29557d87e108b77706eef16f509, so it's more clear what this variable about)